database/tbcd/level/headercache.go

```diff
@@ -47,9 +50,40 @@ func (l *lowIQMap) Get(k *chainhash.Hash) (*tbcd.BlockHeader, bool) {
 	return bh, ok
 }
 
-func lowIQMapNew(count int) *lowIQMap {
+func (l *lowIQMap) Purge(k *chainhash.Hash) {
```

Comment:
if this function is not used, can we delete it?

Reply:
removed